gemma-web/src/test/java/ubic/gemma/web/services/rest/DatasetsRestTest.java

```diff
@@ -41,7 +41,7 @@
     @Before
     public void setUp() {
         // FIXME: this should not be necessary, but other tests are not cleaning up their fixtures
-        expressionExperimentService.remove( expressionExperimentService.loadAll() );
+        expressionExperimentService.removeAll();
```

Comment:
It would be worthwhile to scan the code for such patterns and use the transactional `removeAll` instead.